clients/browser/index.html

```diff
@@ -427,11 +427,11 @@
 
             var el = document.createElement('div');
             var date = new Date(head.timestamp * 1000);
-            var html = `<div><b>${date}</b> hash=<hash>${$.blockchain.headHash.toBase64()}</hash>, `
+            var html = `<div><b>${date}</b> hash=<hash>${$.blockchain.headHash}</hash>, `
                 + `miner=<hash>${head.minerAddr}</hash>, difficulty=${head.difficulty}</div>`;
 
             for (var tx of head.transactions) {
-                var senderAddr = 'TODO'; //await tx.getSenderAddr();
+                var senderAddr = await tx.getSenderAddr();
```

Comment:
Ah, roger. I believed the async/await structure where basically the same thing as Promises, but now I just found out that Promises are separate and have wider support (Edge also), so yes, I agree that It makes sense to use Promises. I will rework this.